spel/minimal-linux.json

```diff
@@ -283,6 +356,7 @@
                 "SPEL_EPELRELEASE={{ user `spel_epel7release` }}",
                 "SPEL_EPELREPO={{ user `spel_epelrepo` }}",
                 "SPEL_EXTRARPMS={{ user `spel_extrarpms` }}",
+                "SPEL_FSTYPE={{ user `spel_fstype` }}",
```

Comment:
Instead of exposing a user variable, maybe you can use the `sed` function creatively on the `build_name`? That might address your _Note2_...

Reply:
I can test with AMIgen7's current state, but I'm pretty much assuming that I would need to make another mod to AMIgen7 to support this request.

Reply:
Alright, looks like when I changed AMIgen7, I did it in a way that _should_ cope with `SPEL_FSTYPE` being null.

~~~
$ cat /tmp/naff.sh
#!/bin/bash
FSTYPE="${FSTYPE:-ext4}"

echo "FS-type: ${FSTYPE}"

$ echo $( export FSTYPE="" &&  bash /tmp/naff.sh )
FS-type: ext4
~~~

Guess we'll see, shortly.

Reply:
You could always force the default FSTYPE in spel's amigen7-build.sh anyway

Reply:
Unfortunately, the `sed` function is pretty primitive: doesn't really offer a way working for `SPEL_FSTYPE` without changing the builder name of the non-XFS builder.

Withdrawing the PR.